### PR TITLE
remove unused lib

### PR DESCRIPTION
**Pull request by eaglelaw**

### CODE REVIEW

**Comment by eaglelaw** [comment]
@xiaoxiang781216 , previous PR seems doesn't working，please review this

**Comment by xiaoxiang781216** [comment]
@eaglelaw please remove the middle merge patch.

**Comment by xiaoxiang781216** [comment]
@eaglelaw could you merge all change into one patch?

File: arch/arm/src/phy62xx/Make.defs [hunk 1]
```diff
@@ -20,12 +20,11 @@
 
 include armv6-m/Make.defs
 
-CMN_ASRCS := $(filter-out arm_exception.S,$(CMN_ASRCS))
+#CMN_ASRCS := $(filter-out arm_exception.S,$(CMN_ASRCS))
```
Comment (xiaoxiang781216) [inline arch/arm/src/phy62xx/Make.defs:23]: remove the unused code

File: arch/arm/src/phy62xx/Make.defs [hunk 2]
```diff
@@ -60,6 +59,7 @@ INCLUDES += $(shell $(INCDIR) "$(CC)" $(ARCH_SRCDIR)$(DELIM)chip$(DELIM)ble)
 #INCLUDES += $(shell $(INCDIR) "$(CC)" $(ARCH_SRCDIR)$(DELIM)chip$(DELIM)ble$(DELIM)include)
 #INCLUDES += $(shell $(INCDIR) "$(CC)" $(ARCH_SRCDIR)$(DELIM)chip$(DELIM)osal$(DELIM)include)
 
+CFLAGS += -ffunction-sections
```
Comment (xiaoxiang781216) [inline arch/arm/src/phy62xx/Make.defs:62]: remove done in armv6-m/Make.defs

File: arch/arm/src/phy62xx/Make.defs [hunk 3]
```diff
@@ -78,37 +78,31 @@ CFLAGS += -Wno-unused-but-set-variable
...
-			curl -L -o libphy62xxble.a http://www.phyplusinc.com/phyplus/libphy62xxble.a; \
-			mkdir -p ../../../staging; \
-			cp -a libphy62xxble.a ../../../staging; \
-		 else \
-			echo "############file exist############"; \
-		fi \
-	fi
-
-context:: .buildlib
+LDFLAGS += -Map="../../../phyplus_build.map"
```
Comment (xiaoxiang781216) [inline arch/arm/src/phy62xx/Make.defs:81]: remove done by armv6-m/Make.defs

File: arch/arm/src/phy62xx/Make.defs [hunk 4]
```diff
@@ -78,37 +78,31 @@ CFLAGS += -Wno-unused-but-set-variable
...
 CHIP_CSRCS += flash.c
-CHIP_CSRCS += clock.c 
+CHIP_CSRCS += clock.c
+CHIP_CSRCS += gpio.c
+CHIP_CSRCS += pwrmgr.c
+CHIP_CSRCS += uart.c
+CHIP_CSRCS += my_printf.c
+CHIP_CSRCS += phy6222_patch.c
+else
+#EXTRA_LIBPATHS += -L$(TOPDIR)/../apps/phy6222/bbb_sdk/lib
```
Comment (xiaoxiang781216) [inline arch/arm/src/phy62xx/Make.defs:93]: remove the comment

File: arch/arm/src/phy62xx/Make.defs [hunk 5]
```diff
@@ -119,7 +113,9 @@ EXTRA_LIBS += -lethermind_utils
 endif
 
 ifdef CONFIG_MIJIA_APIS
-EXTRA_LIBPATHS += -L$(TOPDIR)/../apps/phy6222/bbb_sdk/components/xiaomi/libs/mesh_auth/debug/
-EXTRA_LIBS += -lmesh-auth-cortex-m0plus-debug
+#EXTRA_LIBPATHS += -L$(TOPDIR)/../apps/phy6222/bbb_sdk/components/xiaomi/libs/mesh_auth/debug/
+EXTRA_LIBPATHS += -L$(TOPDIR)/../apps/vendor/phyplus/phy6222_sdk/components/xiaomi/libs/mesh_auth/debug/
+#EXTRA_LIBS += -lmesh-auth-cortex-m0plus-debug
```
Comment (xiaoxiang781216) [inline arch/arm/src/phy62xx/Make.defs:118]: remove

File: arch/arm/src/phy62xx/Make.defs [hunk 6]
```diff
@@ -119,7 +113,9 @@ EXTRA_LIBS += -lethermind_utils
 endif
 
 ifdef CONFIG_MIJIA_APIS
-EXTRA_LIBPATHS += -L$(TOPDIR)/../apps/phy6222/bbb_sdk/components/xiaomi/libs/mesh_auth/debug/
-EXTRA_LIBS += -lmesh-auth-cortex-m0plus-debug
+#EXTRA_LIBPATHS += -L$(TOPDIR)/../apps/phy6222/bbb_sdk/components/xiaomi/libs/mesh_auth/debug/
```
Comment (xiaoxiang781216) [inline arch/arm/src/phy62xx/Make.defs:116]: remove

File: arch/arm/src/phy62xx/phy62xx_exception.S [hunk 7]
```diff
@@ -1,22 +1,3 @@
-/***************************************************************************
```
Comment (xiaoxiang781216) [inline arch/arm/src/phy62xx/phy62xx_exception.S:1]: keep the copyright

**Comment by xiaoxiang781216** [comment]
Don't change the file permission from 100644 to 100755

**Review by xiaoxiang781216 — APPROVED** [review]

**Review by xiaoxiang781216 — APPROVED** [review]

**Pull request merged**